Project.toml

```
@@ -15,7 +16,8 @@ Unicode = "4ec0a83e-493e-50e2-b9ac-8f72acf5a8f5"
 [compat]
 CategoricalArrays = "0.8, 0.9, 0.10"
 CodecZlib = "0.4, 0.5, 0.6, 0.7"
-DataFrames = "0.21, 0.22, 1.0"
+DataAPI = "1.11"
+DataFrames = "1.4"
```

DataAPI.jl for consistency as 1.11 should not be used
```suggestion
DataAPI = "1.12.0"
DataFrames = "1.4.0"
```